numba/core/typing/templates.py

```diff
@@ -387,6 +387,15 @@ def get_template_info(self):
         }
         return info
 
+    def get_cache_deps_info(self, sig):
```

Comment:
Could you add type information to this function?

Reply:
added